plugin-descriptor.properties

```diff
@@ -0,0 +1,8 @@
+name=skroutz-greek-stemmer
+description=Greek stemmer customized for the needs of www.skroutz.gr
+version=0.0.13-snapshot
```

Comment:
Although plugin `version` is updated by the maven release plugin, the version of this file will not change. IMO, this version should have the published version of the plugin(`0.0.13`)